Src/ILGPU/ILGPU.csproj

```diff
@@ -145,6 +150,16 @@
           <AutoGen>True</AutoGen>
           <DependentUpon>IntrinsicMatchers.tt</DependentUpon>
         </Compile>
+        <Compile Update="Runtime\ExchangeBuffer.cs">
```

Comment:
ISSUE: Line 153 to 157 appears to be obsolete, from when `ExchangeBuffer.cs` was changed into `ExchangeBuffer.tt`. Please remove.